packages/office-ui-fabric-react/src/components/Persona/Persona.Props.ts

```diff
@@ -103,11 +103,12 @@ export interface IPersonaProps extends React.HTMLProps<Persona> {
 export enum PersonaSize {
   tiny = 0,
   extraExtraSmall = 1,
-  extraSmall = 2,
-  small = 3,
-  regular = 4,
-  large = 5,
-  extraLarge = 6
+  size28 = 2,
```

Comment:
We can't change any of the existing enums, as some apps rely on the numbers rather than names and this would break them. The new size should be added do the bottom as number 7.